packages/orbit-components/src/ButtonMobileStore/index.tsx

```diff
@@ -41,16 +30,17 @@ const ButtonMobileStore = ({
   };
 
   return (
-    <StyledButtonMobileStore
+    <a
+      className="inline-block h-[40px]"
       href={href}
       target="_blank"
-      rel="noopener"
+      rel="noopener noreferrer"
       onClick={onClickHandler}
       data-test={dataTest}
       id={id}
     >
-      <img srcSet={getSrc(type, lang)} height={HEIGHT} alt={alt} />
-    </StyledButtonMobileStore>
+      <img srcSet={getSrc(type, lang)} height="40px" alt={alt} />
```

Comment:
It has not effect, img has auto height from the preflight. The parent container has fix height, so I guess it's not needed :) 

```suggestion
      <img srcSet={getSrc(type, lang)}  alt={alt} />
```